### ISSUE TITLE
monodepth2 added ARN

### ISSUE THREAD

**Issue by Be997398715** [JiawangBian/SC-SfMLearner-Release#94]
Thank you for your great project. I try to train monodepth2 with ARN module for tum dataset(most of sequences, every sequence for per 5 frame sampling, h*w=640*480, mono train mode with gt_depth), and I find the old monodepth2 could not handle this dataset well, cause every step the loss is 0, while monodepth2 with ARN could work a litter(loss could drop from 0.2 to 0.05 by ARN module loss funtion), where should I pay attention to when add ARN to monodepth2 as a seperate module in mono train mode?

**Comment by JiawangBian** [comment]
I didn't try it directly on the monodepth2 framework. I suggest that you use our PyTorch lightning re-implementation [https://github.com/JiawangBian/sc_depth_pl](https://github.com/JiawangBian/sc_depth_pl), where we have added the features of monodepth2 (minimum loss, auto-mask) into our baseline.

**Comment by Be997398715** [comment]
> 

so sc_depth_pl includes all features in monodepth2 right? I will try later, thanks for your reply:)
